src/Result/Invoice.php

```diff
@@ -61,16 +72,38 @@ public function getExpirationTime(): int
         return $this->getData()['expirationTime'];
     }
 
-    public function getMonitoringTime(): int
```

Comment:
Not sure why or what this is, on my API docs `monitoringTime` and èxpirationTime` still exist and there is no `monitoringExpiration` field on the invoice object? Also would be a breaking change.

Reply:
Yeah the API docs are wrong, this is the actual key that is returned by GF

Reply:
That's bad then, will check

Reply:
I checked both create invoice and get invoice and both return the mentioned `monitoringExpiration` field. But this is additional to the existing two fields. So I think this field is more about how long BTCPay internally monitors the invoice to send webhooks? 

```
    "status": "New",
    "additionalStatus": "None",
    "monitoringExpiration": 1667150955,
    "expirationTime": 1667064555,
    "createdTime": 1667064435,
```


I think you can add a getter for this new field but keep the other two as is.

Reply:
Created an issue for it https://github.com/btcpayserver/btcpayserver/issues/4248

Reply:
Not sure I understand, are you saying I should re-add getMonitoringTime even though it's not returned in the response?

Reply:
Damn must have been late. I was completely confused and you did it right, the created and expiration time are kept and monitoring renamed to the proper property value. Sorry.

Reply:
lol no worries you are still catching so many good things, appreciate the review!

I think we're good to merge this, I'll start working on the next client.